great_expectations/metrics/__init__.py

```diff
@@ -0,0 +1 @@
+from .metric import Metric
```

Comment:
Do we want to have all metrics available at `gx.metrics.*`. That way people don't need to reach into `column_map` to find their metric.

Reply:
With the current syntax, the issue with that is that it might not be clear what the metric represents from the import:

```
from great_expectations.metrics.column_map import Between
```
vs the proposed:
```
from great_expectations.metrics import Between
```

I'm pretty sure based on our current metric names there wouldn't be any namespace collisions though (all imports `from great_expectations.metrics` would be distinct in the proposed import.